GoMeet/src/main/webapp/js/meeting-location-dao.js

```diff
@@ -23,8 +23,8 @@ class MeetingLocationDAO {
     params.append('lat', lat);
     params.append('lng', lng);
     params.append('note', note);
-    let result = await fetch(this.storingEndPoint, {method: 'POST', body: params})
-      .then((response) => response.json());
+    let response = await fetch(this.storingEndPoint, {method: 'POST', body: params});
+    let result = await response.json();
```

Comment:
Why did you change from arrow function?

Reply:
I received a comment on a previous PR that using two await statements is cleaner.

Reply:
Oh my bad! - I thought we were told to preferentially use arrow functions for these async calls

Reply:
Haha nw!